### ISSUE TITLE
error on release, but not in debug or dev: any way to have the exact same behavior as release in dev ?

### ISSUE THREAD

**Issue by wighawag** [tauri-apps/tauri#1679]
**Describe the bug**
on dev and debug I see that the app load the files from the specified server url but in release it use the bundled files

I have an issue where In release, the app is failing to load files, while it works fine on dev mode. I suppose there are some url path issues somewhere

Is there a way to run dev without server so that it behave exactly like in release ?

**Comment by nothingismagick** [comment]
There is no server anymore. Are you certain you are using rhe beta? Can you provide the results of `yarn tauri info`?

**Comment by lucasfernog** [comment]
If there's issues loading paths, we need to track them and fix it. Can you share an example we can use to reproduce?

**Comment by lucasfernog** [comment]
There is a way to run dev without server. Specify a path to the directory with a index.html file on tauri.conf.json > build > devPath.

**Comment by wighawag** [comment]
Hi @nothingismagick and @lucasfernog thanks for your quick replies

My project is not public yet, I ll try to reproduce it with a simpler project once I get some time.

this is my tauri info :

```
Operating System - Windows, version 10.0.19042 X64

Node.js environment
  Node.js - 14.16.0
  @tauri-apps/cli - 1.0.0-beta-rc.4
  @tauri-apps/api Not installed

Global packages
  npm - 6.14.11
  yarn - 1.22.5

Rust environment
  rustc - 1.51.0
  cargo - 1.51.0

App directory structure
/.svelte
/.vscode
/build
/node_modules
/src
/src-tauri
/static

App
  tauri.rs - 1.0.0-beta-rc.4
build-type - bundle
CSP - default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'
distDir - ../build
devPath - http://localhost:8080
```

If I set devPath to ` "devPath": "../build/index.html",` 

and do `tauri dev` I get that in the console:

```
 app:spawn Running "C:\dev\wighawag\conquest-eth\node_modules\.pnpm\@tauri-apps+cli@1.0.0-beta-rc.4\node_modules\@tauri-apps\cli\bin\tauri-cli.exe tauri dev" +0ms

   Compiling app v0.1.0 (C:\dev\wighawag\conquest-eth\web\src-tauri)
    Finished dev [unoptimized + debuginfo] target(s) in 4.33s
     Running `target\debug\app.exe`
AssetNotFound("index.html")
```

and on the app I get a page with the following content:

```
File not found

It may have been moved, edited, or deleted.

ERR_FILE_NOT_FOUND
```

**Comment by wighawag** [comment]
Oh wait, never mind, I misread you @lucasfernog if I set devPath to `../build` I get the same issue as on release

Thanks I ll investigate more what is the issue and report back

**Comment by nothingismagick** [comment]
What framework are you using, and are you using hash or history mode?

**Comment by lucasfernog** [comment]
> What framework are you using, and are you using hash or history mode? 

Seems like svelte from the file structure.

**Comment by wighawag** [comment]
yes svelte + svelte-kit

The issue seems to be about svg images. they do show up as broken but on the network tab I see 200 response
weirdly other svg image, loaded from web url (and not `file://custom-protocol-taurilocalhost/images/...`) works fine

**Comment by lucasfernog** [comment]
SVG loading was fixed on [wry 0.9 release](https://github.com/tauri-apps/wry/blob/dev/CHANGELOG.md#090). It'll be available on the next release.
